CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 0.1.1
 =====
 ## [Bugfix release v0.1.1](https://github.com/rogalmic/vscode-bash-debug/releases/tag/v0.1.1)
+- new launch parameters: program, args, cwd, pathBash, pathBashdb, pathCat, pathMkfifo, pathPkill, trace
```

Comment:
Could you enclose variables in backticks for visibility (example: compare vscode-python's [raw markdown](https://raw.githubusercontent.com/Microsoft/vscode-python/release-2018.2/CHANGELOG.md) and one on vscode) ? like:
``
`pathBash`
``